### PR TITLE
Adds vali instructions paper to vali set

### PR DESCRIPTION
**Pull request by Walarks**
## About The Pull Request
Adds instructions about the Vali module so people know what it generally does.

## Why It's Good For The Game
People have to ask what it does less.

## Changelog
:cl:
qol: added instructions FAQ paper to vali set
/:cl:

### CODE REVIEW

**Comment by Chaznoodles** [comment]
If you need to give people an instruction manual, it's probably too complicated.

**Comment by Walarks** [comment]
> If you need to give people an instruction manual, it's probably too complicated.

People need instructions for a lot of mechanics in the game. Issue is that there are only so many ways to give hints on what to do with stuff compared to most other games.

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -434,3 +434,19 @@ then, for every time you included a field, increment fields. */
 
 /obj/item/paper/crumpled/bloody/csheet
 	info = "<b>Character Sheet</b>\n\nKorbath the Barbarian\nLevel 6 Barbarian\nHitpoints: 47/93\nSTR: 18\nDEX: 14\nCON: 16\nINT: 8\nWIS: 11\nCHA: 15\n\n<B>Inventory:</b>\nGreatsword +3 \nChain M--\n\n\nThe rest of the page is covered in smears of blood."
+
+/obj/item/paper/chemsystem
+	name = "Vali manual"
+	info = {"<b>How to use the Vali system</b><BR>
+	The Vali system uses green blood to empower reagents inside of your body.
+	This green blood is collected by attacking with a connected weapon.<BR>
+	<BR>
```
Comment (SplinterGP) [inline code/modules/paperwork/paper.dm:443]: Do you need BR? i believe you can just use a newline for that and it works fine, also BR is lowercase

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Smudged** [comment]
> If you need to give people an instruction manual, it's probably too complicated.

The instruction manual is if anything in the codex, the paper will just smooth the process

File: code/modules/paperwork/paper.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/paperwork/paper.dm:443]: BR is fine

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**